Felienne/hedy
# Removed Variable view in level 1

**Description**
Removing the variable view in level by Marleen and Onno's request, since there are no variables yet in level 1.

**Fixes issue #2735 **


**How to test**
Go to level 1, make sure that you are not able to click on the label button. Level 2 and beyond should have the variable view button. 

![image](https://user-images.githubusercontent.com/33595265/168069785-de71c511-0d6f-4fc4-8ece-a1ed2129b3b6.png)
